app/views/install.scala.txt

```diff
@@ -245,7 +245,7 @@ if [[ -z "$ARCHIVE_OK" ]]; then
 	echo "* easy sign up: https://slack.sdkman.io/"
 	echo "* report on channel: https://sdkman.slack.com/app_redirect?channel=user-issues"
 	rm -rf "$SDKMAN_DIR"
-	exit
+	exit 2
```

Comment:
Is there a reason why this is 2, not 1?

Reply:
Just to differentiate between missing dependencies and error during file decompression

Reply:
Just noticed Travis has failed, I'll take a look at the error tonight

Reply:
I think it's trying to publish an artifact to docker hub. I need to exclude the publish phase on PRs.